src/applications/check-in/components/layout/Footer.jsx

```diff
@@ -47,15 +58,23 @@ const Footer = ({ router, isPreCheckIn }) => {
       >
         {t('need-help')}
       </h2>
-      {showTravelHelp() ? (
-        <div data-testid="check-in-message">
-          <HelpBlock travel />
-        </div>
-      ) : (
+      {showTravelHelp() &&
```

Comment:
I feel like we could simplify this stuff more. I would recommend getting rid of the isPreCheckIn prop, and rely on the app value from redux. Then I think these helper functions could get named better like `showDayOfTravelHelp()` and `showTravelClaimHelp()`

Reply:
I didn't particularly like it either, but didn't see a vastly superior solution in my work that also kept the readability up